lib/commands/waitForExist.js

```diff
@@ -65,7 +65,9 @@ module.exports = function waitForExist(selector, ms, reverse) {
 
                 var now = new Date().getTime();
                 if(now - start > ms) {
-                    return cb(null, false);
```

Comment:
I'm not sure I understand.  If you are waiting for an element to not exist (reverse = true), then it would be caught by the (reverse && !isExistent) condition above:
```javascript
  var isExistent = res.value.length > 0;
  if ((!reverse && isExistent) || (reverse && !isExistent)) {
    return cb(null, true);
  }
```
If you are expecting an element to eventually exist, I would think you would want an error to be thrown if it times out.  I'm not sure under what condition this function should time out and be successful.

---
My understanding of this function is that you either are expecting something to exist (reverse=false) or something to not exist (reverse=true) and
* If the condition specified is matched before the timeout, resolve
* If the condition specified is not matched before the timeout, throw an error